src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1541,6 +1541,60 @@ public function testEnvCaseInsensitiveOnWindows()
         }
     }
 
+    public function testMultipleCallsToProcGetStatus()
+    {
+        $process = $this->getProcess('echo foo');
+        $process->start(function () use ($process) {
+            $process->isRunning();
+        });
+        while ($process->isRunning()) {
+            usleep(1000);
+        }
+        $this->assertSame(0, $process->getExitCode());
+    }
+
+    public function testFailingProcessWithMultipleCallsToProcGetStatus()
+    {
+        $process = $this->getProcess('exit 123');
+        $process->start(function () use ($process) {
+            $process->isRunning();
+        });
+        while ($process->isRunning()) {
+            usleep(1000);
+        }
+        $this->assertSame(123, $process->getExitCode());
+    }
+
+    /**
+     * @group time-sensitive
```

Comment:
Not sure this is needed since we cannot mock the time for this one, is it? (same below)

Reply:
I've added `@group slow` instead, just to keep track of it due to the arbitrary wait.